### DISCUSSION TITLE
dompdf worked and now it doesn't!

### DISCUSSION THREAD

**Discussion by rodwolford** [dompdf/dompdf#2712] [General]
A programmer used dompdf for a project of mine. It was working fine until just two days ago. I am trying to figure out what is wrong. Here is the error message I am getting. 

_Warning: file_put_contents(custom/certificate/1234.pdf): failed to open stream: No such file or directory in /var/www/vhosts/fofcom.com/httpdocs/post/custom/make_pdf.php on line 37_ 

I have attached a zip file of the make_pdf.php [make_pdf.zip](https://github.com/dompdf/dompdf/files/7782601/make_pdf.zip). You will see I have substituted fixed variables for unique_id, ; username, and quiz_name in an attempt to preclude the possibility of failure due to bad data. 

Also, this is an image of the directory in which all referenced files exist, including make_pdf.php, dompdf, and the folder to which the pdf is written (certificate) ![image](https://user-images.githubusercontent.com/96757753/147525294-d54beca3-fa7d-49f5-9896-45a18fcaddda.png)

As I say, this was working until just about three days ago, then it stopped making pdfs. I hope someone can help and I expect I can learn something from this process. I must say that dompdf seemed like the perfect solution to my needs.

Thanks,

Rod

**Comment by bsweeney** [comment]
`file_put_contents("custom/certificate/{$uniq_id}.pdf", $pdf);` should probably be `file_put_contents("certificate/{$uniq_id}.pdf", $pdf);` since the "custom" path is the current directory for make_pdf.php. Unless this is running in the context of a script one level up.

As for why the current script isn't not working now where it was before, it's not related to Dompdf itself so ¯\\_(ツ)_/¯

**Comment by rodwolford** [comment]
Thanks for the response. I came to the same conclusion that the path was wrong. Funny thing is that I found other path statements that were wrong too. As you say ¯\\_(ツ)_/¯! Anyway thanks a lot for looking at my issue. I really like Dompdf and expect to get much more use from it in the months ahead.

**Comment by markkimsal** [comment]
Are you out of drive space?
